dds_web/api/project.py

```diff
@@ -967,10 +967,16 @@ def get(self):
         project = dds_web.utils.collect_project(project_id=project_id)
         dds_web.utils.verify_project_access(project=project)
 
+        # if current user Researcher, show unit name instead of creator name
+        if auth.current_user().role not in ["Super Admin", "Unit Admin", "Unit Personnel"]:
+            project_creator = project.responsible_unit.name
```

Comment:
Show the external display name instead, the name is mostly for DDS internal purposes, the external name should be displayed in emails etc to the users
```suggestion
            project_creator = project.responsible_unit.external_display_name
```